cds-snc/covid-alert-portal
# Add relationship between location and registrant

# Summary | Résumé

Creates a relationship from Location -> Registrant.

We had not been capturing this relationship previously. There has been some discussion that during the Pilot we may need to reach out to registrants for follow-up information, and may need to search for them based on criteria in the Location model (ie, category).

This PR adds a foreign-key relationship to the Location model (registrant_id). This relationship was created as a nullable column to keep it flexible.

## To test
- Pull down this PR
- Run migrations `python manage.py migrate`
- Browse the database and note the new `registrant_id` column in the `register_location` table
- Go through the process to create a new location using the forms
- After submitting at the end, check the database and notice your new location has a relationship to your Registrant id
